db/migrations/2025_03_25_133231_add_user_id_to_new_strains.py

```diff
@@ -0,0 +1,22 @@
+import sqlalchemy as sql
+
+
+def up(conn):
+    query = """
+        ALTER TABLE Strains
+        ADD COLUMN userUniqueID VARCHAR(100) DEFAULT NULL;
+    """
+    conn.execute(sql.text(query))
```

Comment:
A custom strain will be created by a user and that would allow them to reuse them in the future without re-creating them.